### PR TITLE
Allow shift+click on a profile to open a new window

### LINKED ISSUE #9395: Shift+Click on a profile opens a new window
# Description of the new feature/enhancement

We already ship `Alt+Click` to split a new pane of a profile from the dropdown menu. Now that we have the `newWindow` action, it would be cool if we allowed `Shift+Click` to open a new window of a profile from the dropdown menu.

# Proposed technical implementation details (optional)

### PR DESCRIPTION
**Pull request by PankajBhojwani**
## Summary of the Pull Request
Shift+click on a profile to open a new wt window with that profile. Or, shift+click on the '+' button to open a new wt window with the default profile. 

## PR Checklist
* [x] Closes #9395 
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Documentation updated. If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/terminal) and link it here: #xxx
* [ ] Schema updated.
* [x] I work here

## Validation Steps Performed
Manual testing

### CODE REVIEW

File: src/cascadia/TerminalApp/TerminalPage.cpp [hunk 1]
```diff
@@ -218,6 +223,10 @@ namespace winrt::TerminalApp::implementation
                                      0.5f,
                                      nullptr);
                 }
+                if (shiftPressed && !debugTap)
```
Comment (Don-Vito) [inline src/cascadia/TerminalApp/TerminalPage.cpp:226]: Shouldn't it be esle if? Are we allowing both split pane and new window on alt+shift click?

Comment (PankajBhojwani) [reply src/cascadia/TerminalApp/TerminalPage.cpp]: You are right! Thanks for the catch, fixed

**Review by Nacimota — COMMENTED** [review]
I wonder if the tooltip/s should be updated as well, or would that be making them too wordy?

**Comment by Don-Vito** [comment]
> 
> 
> I wonder if the tooltip/s should be updated as well, or would that be making them too wordy?

I agree with @Nacimota, since we already mention alt+click in tooltip we should probably mention shift+click as well.
Given we put it in a separate line of the tooltip, I believe it is not too wordy.

Alternatively, we could probably show different tooltips given the modifier (but I am not even sure what is the best way to implement such behavior).

**Comment by Nacimota** [comment]
> Alternatively, we could probably show different tooltips given the modifier (but I am not even sure what is the best way to implement such behavior).

You mean instead of the current tip it would say "Split the current tab" when you hold alt? There is something about that which I find appealing, but I can't help but think it takes away from the discoverability a little bit, which I think is really what the tooltip is all about.

I've just tried it with both tips on separate lines as you suggested and I agree that it does not look too bad. I do worry that one can put too much text in there, though.

tooltip

**Comment by Don-Vito** [comment]
> 
> 
> > Alternatively, we could probably show different tooltips given the modifier (but I am not even sure what is the best way to implement such behavior).
> 
> You mean instead of the current tip it would say "Split the current window" when you hold alt, for example? There is something about that which I find appealing, but I can't help but think it takes away from the discoverability a little bit, which I think is really what the tooltip is all about.
> 

I agree that it somewhat reduces the value of the tool tip - it was just a fallback if we believe that the tool tip is too cluttered.

Somewhere in a separate thread we discussed alternatives for providing this information not via tool-tip but through some visible element.

**Review by zadjii-msft — CHANGES_REQUESTED** [review]
This code looks fine, but I suppose I should block over the tooltip text

**Comment by PankajBhojwani** [comment]
Tooltip updated! Thanks for the feedback everyone

**Review by zadjii-msft — APPROVED** [review]

**Review by carlos-zamora — APPROVED** [review]
